### PR TITLE
Provide API endpoint for uploading attachments

### PR DESCRIPTION
**Pull request by jh0ker**
:sparkles:

### CODE REVIEW

File: entrypoint/fb.js [hunk 1]
```diff
@@ -178,3 +179,21 @@ module.exports.push = (event, context, callback = console.log) => {
     })
   })
 };
+
+module.exports.attachment = (event, context, callback) => {
+  const payload = JSON.parse(event.body);
+  console.log(payload);
```
Comment (LizzAy) [inline entrypoint/fb.js]: Is this logging still necessary or was it just for testing?

File: entrypoint/fb.js [hunk 2]
```diff
@@ -178,3 +179,21 @@ module.exports.push = (event, context, callback = console.log) => {
...
+      statusCode: 200,
+      body: JSON.stringify({success: true, message: id}),
+    });
+  }).catch(error => {
+    callback(null, {
+      statusCode: 500,
+      body: JSON.stringify({success: false, message: error}),
+    });
+  });
+};
```
Comment (LizzAy) [inline entrypoint/fb.js]: No new line at end of file

File: entrypoint/fb.js (hunk 1 above)
Comment (jh0ker) [reply entrypoint/fb.js]: I'd prefer to keep it

File: entrypoint/fb.js (hunk 2 above)
Comment (jh0ker) [reply entrypoint/fb.js]: :hankey:

**Pull request merged**